state-chain/pallets/cf-asset-balances/src/lib.rs

```diff
@@ -0,0 +1,353 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![doc = include_str!("../../cf-doc-head.md")]
+
+use cf_chains::{AnyChain, ForeignChain, ForeignChainAddress};
+use cf_primitives::{Asset, AssetAmount};
+use cf_runtime_utilities::log_or_panic;
+use cf_traits::{
+	impl_pallet_safe_mode, AssetWithholding, Chainflip, EgressApi, KeyProvider, LiabilityTracker,
+	ScheduledEgressDetails,
+};
+use frame_support::{
+	pallet_prelude::*, sp_runtime::traits::Saturating, traits::DefensiveSaturating,
+};
+use serde::{Deserialize, Serialize};
+use sp_std::{collections::btree_map::BTreeMap, vec, vec::Vec};
+
+pub use pallet::*;
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+#[cfg(test)]
+mod mock;
+#[cfg(test)]
+mod tests;
+
+pub const PALLET_VERSION: StorageVersion = StorageVersion::new(0);
+
+// TODO: Figure out adequate values for these.
+pub const MAX_ETH_REFUND_PER_EPOCH: usize = 25;
+pub const MAX_ARB_REFUND_PER_EPOCH: usize = 10;
```

Comment:
same here

Reply:
Should be `MAX_ARB_ETH_REFUND_PER_EPOCH` (if we have it at all - see above question)

Reply:
Yes, we need one. @albert-llimos tries to figure out a number and the result will most likely surprise you ^^

Reply:
If we go for 50 as above for Ethereum let's do the same for Arbitrum. The way that Arbitrum's gas consumption works is very weird and I'd rather be on the same side. I know maybe you were expecting we would 10x that number in Arbitrum, sorry to dissapoint.